cobalt-transcribe/cmd/root.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (2019) Cobalt Speech and Language Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+type clientConf struct {
+	Server Server
+}
+
+type Server struct {
+	Address     string        // URL of the server as specified by the command line arguments.
+	ModelID     string        // ID of the ASR model
+	Insecure    bool          // Whether to use insecure connection
+	IdleTimeout time.Duration // Time to wait for a response from the server.
+}
+
+// configuration struct to hold global flags
+var (
+	cConf  clientConf
+	confFn string
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "transcribe",
```

Comment:
saying `transcribe <VERB>` seem weird to me.
```suggestion
	Use:   "transcribe-client",
```